algebra/src/main/scala/endpoints/MethodsAlg.scala

```diff
@@ -0,0 +1,16 @@
+package endpoints
+
+
+trait MethodsAlg {
+
+  type Method
+
+  def Get: Method
```

Comment:
That's intentional, to differentiate from methods I have deleted. I will describe more in general comment to PR.